src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2347,5 +2347,16 @@ public boolean isVideo()
             String extension = getFileExtension();
             return extension != null && VIDEO_EXTENSIONS.contains(extension.toLowerCase());
         }
+
+        /**
+         * Whether or not this attachment is tagged as spoiler,
+         * based on {@link #getFileName()}.
+         *
+         * @return True if this attachment is tagged as spoiler
+         */
+        public boolean isSpoiler()
+        {
+            return getFileName().startsWith("SPOILER_");
```

Comment:
Can we check if this is case sensitive or not?

Reply:
it's case sensitive